R/environment.R

```diff
@@ -206,6 +194,20 @@ generate_docker_file <- function(custom_docker_image = NULL,
   base_dockerfile <- NULL
   base_dockerfile <- paste0(base_dockerfile, sprintf("FROM %s\n",
                                                      custom_docker_image))
+  
+  base_dockerfile <- paste0(base_dockerfile, "RUN conda install -c r -y ",
+                            "r-essentials && conda clean -ay\n")
+  
+  base_dockerfile <- paste0(base_dockerfile, "RUN pip install --no-cache-dir ",
+                            "azureml-defaults\n")
```

Comment:
merge to one docker image layer to save some space. also pin r-essentials to 3.6.0 for now